internal/deb/helpers_test.go

```diff
@@ -0,0 +1,63 @@
+package deb_test
+
+import (
+	"github.com/canonical/chisel/internal/deb"
+	. "gopkg.in/check.v1"
+)
+
+func fakePlatformGoArch(goArch string) (restore func()) {
+	if goArch != "" {
+		saved := deb.PlatformGoArch
+		deb.PlatformGoArch = goArch
```

Comment:
This bit doesn't belong in the public interface. Please have a look at `FakeShellcheckPath(p string)` under `testutil/export_test.go` for an example of how this may be handled nicely.